src/js/filemanager.js

```diff
@@ -351,22 +351,10 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
 	};
 
 	var includeAssets = function(callback) {
-		var primary = [],
```

Comment:
To jakieś stare czyszczenie jest, którego nie zapomniałem wcześniej push-nąć, nie zwracajcie na to uwagi najlepiej